packages/react-sdk/src/user-props/schema.ts

```diff
@@ -11,7 +11,7 @@ const UserPropDb = z.object({
 
 export const UserDbProps = z.array(UserPropDb);
 
-const UserUserProp = z
+export const UserUserProp = z
```

Comment:
This type has changed in `main` in two important ways since I made this PR. Can't use it as is now. Need to figure out how to update

Reply:
@istarkov I've defined Asset in `zod` and moved transform to where we write to DB. Please check.